src/bluesky/callbacks/_mpl_image_cross_section/cross_section.py

```diff
@@ -0,0 +1,469 @@
+import logging
```

Comment:
Given that we dropped all of the rest of the code, we might as well consolidate all of this code into a single `_mpl_image_cross_section.py` ?

Reply:
consolidating now